pxr/usd/usdLux/lightAPI.h

```diff
@@ -359,10 +421,119 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
     // --------------------------------------------------------------------- //
     // NORMALIZE 
     // --------------------------------------------------------------------- //
-    /// Normalizes power by the surface area of the light.
-    /// This makes it easier to independently adjust the power and shape
-    /// of the light, by causing the power to not vary with the area or
-    /// angular size of the light.
+    /// Normalizes the emission such that the power of the light
+    /// remains constant while altering the size of the light, by dividing the
+    /// luminance by the world-space surface area of the light.
+    /// 
+    /// This makes it easier to independently adjust the brightness and size
+    /// of the light, by causing the total illumination provided by a light to
+    /// not vary with the area or angular size of the light.
+    /// 
+    /// Mathematically, this means that the luminance of the light will be
+    /// divided by a factor representing the "size" of the light:
+    /// 
+    /// <center><b>
+    /// L<sub>Scalar</sub> = L<sub>Scalar</sub> / sizeFactor
+    /// </b></center>
+    /// 
+    /// ...where `sizeFactor` = 1 if `normalize` is off, and is calculated
+    /// depending on the family of the light as described below if `normalize`
+    /// is on.
+    /// 
+    /// ### DomeLight / PortalLight:
+    /// 
+    /// For a dome light (and its henchman, the PortalLight), this attribute is
+    /// ignored:
+    /// 
+    /// <center><b>
+    /// sizeFactor<sub>dome</sub> = 1
+    /// </b></center>
+    /// 
+    /// ### Area Lights:
+    /// 
+    /// For an area light, the `sizeFactor` is the surface area (in world
+    /// space) of the shape of the light, including any scaling applied to the
+    /// light by its transform stack. This includes the boundable light types
+    /// which have a calculable surface area:
+    /// 
+    /// - MeshLightAPI
+    /// - DiskLight
+    /// - RectLight
+    /// - SphereLight
+    /// - CylinderLight
+    /// - (deprecated) GeometryLight
+    /// 
+    /// <center><b>
+    /// sizeFactor<sub>area</sub> = worldSpaceSurfaceArea(light)
+    /// </b></center>
+    /// 
+    /// ### DistantLight:
+    /// 
+    /// For distant lights, we first define 𝛳<sub>max</sub> as:
+    /// 
+    /// <center><b>
+    /// 𝛳<sub>max</sub> = clamp(toRadians(distantLightAngle) / 2, 0, 𝜋)
+    /// </b></center>
+    /// 
+    /// Then we use the following formula:
+    /// 
+    /// * <i>if 𝛳<sub>max</sub> = 0:</i>
+    /// <center><b>
+    /// sizeFactor<sub>distant</sub> = 1
+    /// </b></center>
+    /// 
+    /// * <i>if 0 < 𝛳<sub>max</sub> ≤ 𝜋 / 2:</i>
+    /// <center><b>
+    /// sizeFactor<sub>distant</sub> = sin²𝛳<sub>max</sub> ⋅ 𝜋
```

Comment:
Apologies for the dumb questions, but how does this formulation relate to e.g. if you pretend a distant light is a cone section of a sphere?  If you normalize by computing the solid angle of the light vs the solid angle of a light with angle 0.53, the solid angle is 2 * pi * (1 - cos(theta)), or equivalently, 4 * pi * sqr(sin(0.5 * theta)).
https://en.wikipedia.org/wiki/Solid_angle#Cone,_spherical_cap,_hemisphere

... the math looks similar but I'm having trouble drawing the correspondences.  Can you explain what's going on here geometrically?

Reply:
See [the link I posted in response to Curtis, above (page 11)](https://www.cs.cornell.edu/courses/cs5625/2022sp/notes/radiometry-notes.pdf). This is defining it in terms of *projected* solid angle instead of solid angle. This has two nice properties:
1. The light doesn't change brightness when you make the angle bigger (which is the whole point of normalize), see [renders 21-30 here](https://molodowitch.com/luxtest-diffs/2024-07-31.4k-sample-embree.luxtest-1a88c027.usd-6187b7a3.renders-e6fc4200/luxtest.html#distant) comparing first and second columns. First is this normalization, second is Karma, which does solid angle normalization.
2. This makes the light's brightness be defined in units of illuminance, which means we now have a way to ascribe a SI unit to the light.

Reply:
For a geometrical derivation of this formula, see "Property 2" in this explanation I wrote up at one point (because I had the same question):

https://github.com/pmolodo/OpenUSD/blob/pr/usd-lux-update-doc-normalizingDistantLights.pdf/pxr/usd/usdLux/doxygen/normalizingDistantLights.pdf

(There's also a [.md version](https://github.com/pmolodo/OpenUSD/blob/pr/usd-lux-update-doc-normalizingDistantLights.pdf/pxr/usd/usdLux/doxygen/normalizingDistantLights.md), but it's formatted for Doxygen's markdown flavor, not github's, so not all formating works 100%.)